lfs/lfs.go

```diff
@@ -72,10 +76,15 @@ func Environ(cfg *config.Configuration, manifest *tq.Manifest) []string {
 	}
 
 	for _, e := range osEnviron {
-		if !strings.Contains(strings.SplitN(e, "=", 2)[0], "GIT_") {
+		key := strings.SplitN(e, "=", 2)[0]
+		if !strings.Contains(key, "GIT_") {
```

Comment:
Just looking at this with fresh eyes, I wonder if `strings.HasPrefix()` would be more appropriate ... are there environment variables where `GIT_` would not be at the start of the name?

Reply:
Yeah, this is a behavior change, but I believe it's a prudent change to make.